pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -35,13 +36,14 @@ const (
 
 func TestNewPrometheusStatsReporter_negative(t *testing.T) {
 	tests := []struct {
-		name      string
-		errorMsg  string
-		result    error
-		namespace string
-		config    string
-		revision  string
-		pod       string
+		name                    string
+		errorMsg                string
+		result                  error
+		namespace               string
+		config                  string
+		revision                string
+		pod                     string
+		reporterReportingPeriod time.Duration
```

Comment:
Just drop this here and set it to a fixed value maybe? Or add validation of the value and check for error values (like negative durations) in this test.

Reply:
I dropped it and setting to a fixed value.